distribution/registry_unit_test.go

```diff
@@ -41,22 +41,15 @@ func testTokenPassThru(t *testing.T, ts *httptest.Server) {
 	}
 
 	endpoint := registrypkg.APIEndpoint{
-		Mirror:       false,
 		URL:          uri,
-		Official:     false,
 		TrimHostname: false,
```

Comment:
Intentionally kept this one for now, as I'm figuring out where it's used and where it's relevant.